libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/utils/extension/LocaleEx.kt

```diff
@@ -0,0 +1,25 @@
+@file:JvmName("LocaleEx")
```

Comment:
Should we move this into `libnavigation-base` (it's duplicated in `libdirections-offboard`)?

Reply:
`Utils` are duplicated everywhere. I say again that we should remove the `libnavigation-utils` module and have it as a part of base. Or else if we don't want to remove `libnavigation-utils` we add its dependency to `libnavigation-base` so that all other modules can transitively inherit. Please discuss. Not moving it to `libnavigation-base` as of now.

Reply:
Let's start moving duplicated extensions to `libnavigation-base` incrementally. How does that sound?

Reply:
What's the problem in moving the `utils` to `base` at once? It's an `internal` change. If some files are not shared, they may be left out.